ovn_cluster.sh

```diff
@@ -40,7 +40,7 @@ IP_HOST=${IP_HOST:-170.168.0.0}
 IP_CIDR=${IP_CIDR:-16}
 IP_START=${IP_START:-170.168.0.2}
 
-OVN_DB_CLUSTER="${OVN_DB_CLUSTER:-no}"
+OVN_DB_CLUSTER="${OVN_DB_CLUSTER:-yes}"
```

Comment:
on by default, huh? I'm kinda okay with it but we may soon need to think of
a set of defaults for simple/newcomers and one for exercising all the cool stuff
we have in ovn.

Reply:
Oops. That was not the intention. I had modified it locally. I'd prefer cluster to be off by default.